bin/install.sh

```diff
@@ -86,7 +93,21 @@ done
 
 # Copy files
 INSTALL_DIR="$PIER/$SHIP/$DESK"
+
 echo "Attempting to install to path '$INSTALL_DIR'"
+
 cp -rfL ${CHESS_DIR}/* ${INSTALL_DIR}/
 cp -rfL ${DEPS_DIR}/* ${INSTALL_DIR}/
+
+if [ $DOCKET -eq 1 ]; then
+  cp -rfL ${RESOURCE_DIR}/* ${INSTALL_DIR}/
+  DOCKET_FILE="${INSTALL_DIR}/desk.docket-0"
+
+  echo "\n"                         >> $DOCKET_FILE
```

Comment:
Had to change to this to get it working on macOS, otherwise it added the text `\n` in there.

```bash
echo "\n"                         >> $DOCKET_FILE
```

Reply:
Not sure why this didn't come up testing #13. I just tried installing with `-k` on a new fakeship working from the `develop` branch and got this in there too.

```hoon
:~
  title+'Chess'
  info+'Fully peer-to-peer chess over Urbit'
  color+0xff.ffff
  image+'https://peekabooo.icu/images/finmep-chess.svg'
  base+'chess'
  version+[0 0 1]
  license+'GPL3'
  website+'https://github.com/thecommons-urbit/chess'\n
  glob-ames+[~fun 0v0]
==
```

Reply:
So odd - I tested it locally, and my test file didn't have this issue

Reply:
Could be an OS thing. Does it make a difference in Linux if you replace the `"\n"` with `""`?